foundation/foundation-mda/src/main/java/com/boozallen/aiops/mda/ManualActionNotificationService.java

```diff
@@ -326,37 +326,21 @@ public void addLocalResourceTiltFileMessage(final GenerationContext context, fin
      * Checks if there are deployment changes necessary in the Tiltfile for Spark Worker and
      * adds a message if so.
      *
-     * @param context                     the generation context
-     * @param appName                     the application name
-     * @param dockerApplicationArtifactId the docker application artifact ID
-     * @param dockerArtifactId            the docker artifact ID
-     * @param includeHelmBuild            whether to include the helm build in the message
-     * @param includeLatestTag            whether to include the latest tag as a tilt extra tag
+     * @param context the generation context
+     * @param appName the application name
      */
-    public void addSparkWorkerDockerBuildTiltMessage(final GenerationContext context, final String appName, final String dockerApplicationArtifactId,
-                                                     final String dockerArtifactId) {
+    public void addSparkApplicationTiltMessage(final GenerationContext context, final String appName) {
 
         final File rootDir = context.getExecutionRootDirectory();
         if (!rootDir.exists() || !tiltFileFound(rootDir)) {
             logger.warn("Unable to find Tiltfile. Will not be able to direct manual Spark Worker resources to Tiltfile");
         } else {
-            //creates the instructions for the spark worker image
-            DockerBuildParams params = new DockerBuildParams.ParamBuilder().setContext(context)
-                    .setAppName(appName)
-                    .setDockerApplicationArtifactId(dockerApplicationArtifactId)
-                    .setDockerArtifactId(dockerArtifactId)
-                    .setIncludeHelmBuild(false)
-                    .setIncludeLatestTag(true)
-                    .build();
-            addDockerBuildTiltFileMessage(params);
-
             final String tiltFilePath = rootDir.getAbsolutePath() + File.separator + "Tiltfile";
             final String text = "k8s_kind('SparkApplication'";
 
             boolean tiltFileContainsArtifact = existsInFile(tiltFilePath, text);
             if (!tiltFileContainsArtifact && showWarnings(tiltFilePath)) {
-                final String key = getMessageKey("Tiltfile", "spark-worker-docker-build");
-
+                final String key = getMessageKey("Tiltfile", appName);
```

Comment:
Since the value of `appName` is hardcoded within [SparkDockerPomGenerator.java](https://github.com/boozallen/aissemble/pull/506/files#diff-75b8ef1e9b6ff022d38d20d083fa8dd24745affd9836e201b139061c90c1a2f5) to `"pipeline-spark-application"`, think we could just harcode it here instead of passing it as a paremeter.